azure-pipelines.yml

```diff
@@ -1,5 +1,8 @@
 # https://dev.azure.com/spyder-ide/spyder/
 
+variables:
+  RUNNING_AZURE: true  # this is available to all jobs
```

Comment:
There's an `AZURE` variable defined below. Please use that instead of defining a new one.

Reply:
Ok